site/content/en/docs/Guides/access-api.md

```diff
@@ -50,27 +50,28 @@ example from the Kubernetes client.
 The following is an example of a in-cluster configuration, that creates a `Clientset` for Agones
 and then creates a `GameServer`.
 
+{{% feature publishVersion="1.2.0" %}}
```

Comment:
Do we need to feature shortcode this? Once it's in the example repo, it'll be available to the documentation.